doc/hyperfine.1

```diff
@@ -207,6 +213,14 @@ Export the results of a parameter scan benchmark to a markdown table:
 \fBhyperfine\fR \fB\-\-export\-markdown\fR output.md \fB\-\-parameter-scan\fR time 1 5 'sleep {time}'
 .fi
 .RE
+.LP
+Demonstrate when each of \fB\-\-build\fR, \fB\-\-prepare\fR, \fIcmd\fP and \fB\-\-cleanup\fR will run:
+.RS
+.nf
+\fBhyperfine\fR \fB\-\fRL n 1,2 \fB\-r\fR 2 \fB\-\-show-output\fR \fB\-b\fR 'echo build n={n}' \fB\-\fRp 'echo warmup n={n}' 'echo test n={n}' \fB\-\fRc 'echo clean n={n}'
+.fi
+.RE
+.RE
```

Comment:
Nice!

What I would really like to do at some point would be to add an execution flow graph somewhere in the documentation. Ideally, this should be done in graphviz (or similar) to make it easy to modify in the future.